#! /bin/sh /usr/share/dpatch/dpatch-run
## 004_debian_layout_fix.dpatch by  <tfheen@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix the includedir in the Debian layout.

@DPATCH@
diff -urNad apr-util-1.2.7~/config.layout apr-util-1.2.7/config.layout
--- apr-util-1.2.7~/config.layout	2005-02-09 13:18:43.000000000 +0100
+++ apr-util-1.2.7/config.layout	2006-06-15 14:18:42.000000000 +0200
@@ -223,7 +223,7 @@
     libexecdir:    ${exec_prefix}/lib/apr/modules
     mandir:        ${exec_prefix}/share/man
     datadir:       ${exec_prefix}/share/apr
-    includedir:    ${exec_prefix}/include/apr-${APRUTIL_MAJOR_VERSION}
+    includedir:    ${exec_prefix}/include/apr-1.0
     localstatedir: ${prefix}/var/run
     runtimedir:    ${prefix}/var/run
     infodir:       ${exec_prefix}/share/info
